src/detector.c

```diff
@@ -825,7 +834,11 @@ float validate_detector_map(char *datacfg, char *cfgfile, char *weightfile, floa
                     float prob = dets[i].prob[class_id];
                     if (prob > 0) {
                         detections_count++;
-                        detections = (box_prob*)realloc(detections, detections_count * sizeof(box_prob));
+                        box_prob* tmp= (box_prob*)realloc(detections, detections_count * sizeof(box_prob));
```

Comment:
Why do we need `tmp` variable here if we call `error()` function which will terminate the programm?

Reply:
fixed.